internal/service/service.go

```diff
@@ -36,30 +33,6 @@ type Service struct {
 	Port int
 }
 
-//GetVersionJSON returns the request URL and response for the 'version' endpoint.
-func (c Service) GetVersionJSON() (json string, url string, err error) {
```

Comment:
Like with the changes for formatting, this re-factoring should've been committed separately from the new code for Core Data...

Reply:
absolutely. I'll include separate commits next time.